### PR TITLE
Create 2020-03-26-Announcing-Ivanchuk-4.md

### PR DESCRIPTION
**Pull request by mfeifer**
announcement for Ivanchuk-4

### CODE REVIEW

**Comment by mfeifer** [comment]
@cybette please review asap

File: site/_posts/2020-03-26-Announcing-Ivanchuk-4.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+title: Announcing Ivanchuk-4
+author: mfeifer
+date: 2020-03-29 19:12:13 UTC
```
Comment (cybette) [inline site/_posts/2020-03-26-Announcing-Ivanchuk-4.md]: Date is in the future, which is fine but it won't show up on the blog until Friday.

File: site/_posts/2020-03-26-Announcing-Ivanchuk-4.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+### Fixed
+* [V2V] Remove handover from shutdown_vm transition [(#19371)](https://github.com/ManageIQ/manageiq/pull/19371)
+* [V2V] Extend InfraConversionJob timeout [(#19373)](https://github.com/ManageIQ/manageiq/pull/19373)
+* [V2V] Fix progress percentage calculation [(#19375)](https://github.com/ManageIQ/manageiq/pull/19375)
+
+You can download the Ivanchuk-4 release [here](http://manageiq.org/download/).
+
+Here is the changelog:
+[Ivanchuk-3](https://github.com/ManageIQ/manageiq/blob/ivanchuk/CHANGELOG.md)
```
Comment (cybette) [inline site/_posts/2020-03-26-Announcing-Ivanchuk-4.md]: Ivanchunk-4 (not 3)

**Comment by cybette** [comment]
Please also update site/_data/releases.yaml like in https://github.com/ManageIQ/manageiq.org/pull/810

Thanks  @mfeifer!

**Pull request merged**